docs/rules/id-match.md

```diff
@@ -58,6 +58,9 @@ This rule has an object option:
 
 * `"properties": true` requires object properties to match the specified regular expression
 * `"onlyDeclarations": true` requires only `var`, `function`, and `class` declarations to match the specified regular expression
+* `"onlyDeclarations": false` requires all variable names to match the specified regular expression
```

Comment:
nits: `"onlyDeclarations": false` (default) ?

Reply:
I'll separate this code into another commit later.

Reply:
I don't know this. This feature was exist here for about two years. I've checked that `false` is its default value and I don't know why. Default value of `onlyDeclarations` became `false` is also for about two years.